vm/vm_jit.go

```diff
@@ -1,31 +1,270 @@
+// +build evmjit
+
 package vm
 
 import "math/big"
+import "github.com/ethereum/go-ethereum/crypto"
+import "github.com/ethereum/go-ethereum/state"
+
+/*
+#include "../evmjit/libevmjit/interface.h"
+
+#cgo LDFLAGS: -L/home/chfast/go/src/github.com/ethereum/go-ethereum/evmjit/build/libevmjit -levmjit
```

Comment:
That's right, but I don't know better way to integrate cmake building with go build. In the end we can download the shared library from build server.